packages/webapp/src/App.tsx

```diff
@@ -59,7 +59,10 @@ const App = () => {
                     script={[
                         {
                             type: 'text/javascript',
-                            innerHTML: `!function (t) { if (window.ko) return; window.ko = [], ["identify", "track", "removeListeners", "on", "off", "qualify", "ready"].forEach(function (t) { ko[t] = function () { var n = [].slice.call(arguments); return n.unshift(t), ko.push(n), ko } }); var n = document.createElement("script"); n.async = !0, n.setAttribute("src", "https://cdn.getkoala.com/v1/${globalEnv.publicKoalaKey}/sdk.js"), (document.body || document.head).appendChild(n) }();`
+                            innerHTML: `
+                                window.koalaSettings = { host: 'https://bluegrass.nango.dev' };
+                                !function(t){var k="ko",i=(window.globalKoalaKey=window.globalKoalaKey||k);if(window[i])return;var ko=(window[i]=[]);["identify","track","removeListeners","on","off","qualify","ready"].forEach(function(t){ko[t]=function(){var n=[].slice.call(arguments);return n.unshift(t),ko.push(n),ko}});var n=document.createElement("script");n.async=!0,n.setAttribute("src","https://shoegaze.nango.dev/v1/pk_799998f91877afef040206962ce1af164de9/sdk.js"),(document.body || document.head).appendChild(n)}();
+                            `
```

Comment:
you should keep `${globalEnv.publicKoalaKey}` and add a way to parametrize the domain I guess

Reply:
Oh yep, missed that in the super long template string there.

Reply:
@bodinsamuel see 6f13eb81c